packages/flutter/lib/src/semantics/semantics.dart

```diff
@@ -2210,6 +2228,11 @@ class SemanticsNode with DiagnosticableTreeMixin {
   /// Whether this node currently has a given [SemanticsFlag].
   bool hasFlag(SemanticsFlag flag) => _flags & flag.index != 0;
 
+  /// A description used for identifying this node during UI automation testing.
```

Comment:
This can reuse doc from semanticsproperty too

Reply:
How?

Reply:
You shouldn't need to copy-paste. Look for `@template` and `@macro` in the code for examples of automatic copying of docs.